eng/scripts/prepare_chat_tests.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/python3
+
+"""
+prepare_chat_tests.py
+
+1) Run `pip install azure.communication.administration` prior to running.
+
+2) Set the `AZURE_COMMUNICATION_CS` environment variable
+
+3) Run the script.
+
+4) Copy the values to the AzureCommunicationChat scheme's testing environment variables and set `TEST_MODE` to
+   "record".
+"""
+
+import os
+from azure.communication.administration import CommunicationIdentityClient
+
+connection_string = os.environ['AZURE_COMMUNICATION_CS']
+identity_client = CommunicationIdentityClient.from_connection_string(connection_string)
+
+items = { key:val for (key, val) in (x.split('=', 1) for x in connection_string.split(';')) }
+endpoint = items['endpoint']
+
+user1 = identity_client.create_user()
+user2 = identity_client.create_user()
+
+token2 = identity_client.issue_token(user2, scopes=["chat"]).token
+
+print('\n== AZURE_COMMUNICATION_ENDPOINT ==')
+print(endpoint)
+
+print('\n== AZURE_COMMUNICATION_USER_ID_1 ==')
+print(user1.identifier)
+
+print('\n== AZURE_COMMUNICATION_USER_ID_2 ==')
+print(user2.identifier)
+
+print('\n== AZURE_COMMUNICATION_TOKEN ==')
+print(token2)

```

Comment:
I think we should return the token for user1 instead of user2

In the tests the initial user (whose token we use to create the ChatClient) is meant to be user1, and for adding participants we use user2. If we provide the token for user2 it means the initial user is actually user2 & the add participants tests need to be swapped to add user1 instead

Reply:
I used token2 because I didn't actually see any Chat tests that used USER_ID_1. I'll update it.

Reply:
Updated to use token1.